apiclient/account.go

```diff
@@ -227,3 +238,95 @@ func (c *HTTPclient) GetTransfers(from common.Address, page, pageSize int) ([]*i
 	}
 	return transfers, nil
 }
+
+// SetSik function allows to update the Secret Identity Key for the current
+// HTTPClient account. To do that, the function requires a secret user input.
+func (c *HTTPclient) SetSik(secret []byte) (types.HexBytes, error) {
```

Comment:
secret is not used :/

I'd export two methods, this might be useful for testing purposes:

- SetSik()
- SetCustomSik(sik []byte)

The first can call the second.